docs/docsite/rst/user_guide/playbooks_prompts.rst

```diff
@@ -12,27 +12,29 @@ in a push-script.
 
 Here is a most basic example::
 
-    ---
-    - hosts: all
-      remote_user: root
+      ---
```

Comment:
Why the extra 2 spaces, having 4 spaces as the indent is enough for RST to put this in a codeblock.

Reply:
Fixed